data/crac-file/crac-io-json/src/test/java/com/farao_community/farao/data/crac_io_json/RoundTripUtil.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright (c) 2020, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.farao_community.farao.data.crac_io_json;
+
+import com.farao_community.farao.data.crac_impl.SimpleCrac;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.ObjectWriter;
+import com.fasterxml.jackson.databind.module.SimpleModule;
+import com.fasterxml.jackson.datatype.jdk8.Jdk8Module;
+
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.UncheckedIOException;
+
+import static com.powsybl.commons.json.JsonUtil.createObjectMapper;
+
+/**
+ * @author Joris Mancini {@literal <joris.mancini at rte-france.com>}
+ */
+public final class RoundTripUtil {
+    private RoundTripUtil() {
+
+    }
+
+    static <T> T roundTrip(T object, Class<T> objectClass) {
```

Comment:
can you add a small documentation explaining what this method does?

Reply:
sure !